remove unused include of termio.h

This header is not provided by musl, leading to build failures.

Patch status: accepted upstream.

Signed-off-by: Ricardo Martincoski <ricardo.martincoski@datacom.ind.br>
---
diff -purN atop-2.2-3.orig/showgeneric.c atop-2.2-3/showgeneric.c
--- atop-2.2-3.orig/showgeneric.c	2016-07-19 22:07:43.275324499 -0300
+++ atop-2.2-3/showgeneric.c	2016-07-19 22:08:16.171453283 -0300
@@ -268,7 +268,6 @@ static const char rcsid[] = "$Id: showge
 #include <errno.h>
 #include <fcntl.h>
 #include <string.h>
-#include <termio.h>
 #include <unistd.h>
 #include <stdarg.h>
 #include <curses.h>
diff -purN atop-2.2-3.orig/showlinux.c atop-2.2-3/showlinux.c
--- atop-2.2-3.orig/showlinux.c	2016-07-19 22:07:43.275324499 -0300
+++ atop-2.2-3/showlinux.c	2016-07-19 22:08:20.979472459 -0300
@@ -274,7 +274,6 @@ static const char rcsid[] = "$Id: showli
 #include <string.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <termio.h>
 #include <unistd.h>
 #include <stdarg.h>
 #include <curses.h>
diff -purN atop-2.2-3.orig/showprocs.c atop-2.2-3/showprocs.c
--- atop-2.2-3.orig/showprocs.c	2016-07-19 22:07:43.275324499 -0300
+++ atop-2.2-3/showprocs.c	2016-07-19 22:08:25.083488900 -0300
@@ -94,7 +94,6 @@ static const char rcsid[] = "$Id: showpr
 #include <string.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <termio.h>
 #include <unistd.h>
 #include <stdarg.h>
 #include <curses.h>
diff -purN atop-2.2-3.orig/showsys.c atop-2.2-3/showsys.c
--- atop-2.2-3.orig/showsys.c	2016-07-19 22:07:43.275324499 -0300
+++ atop-2.2-3/showsys.c	2016-07-19 22:08:29.099505047 -0300
@@ -80,7 +80,6 @@ static const char rcsid[] = "XXXXXX";
 #include <string.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <termio.h>
 #include <unistd.h>
 #include <stdarg.h>
 #include <curses.h>
